homework/src/App.css

```diff
@@ -0,0 +1,63 @@
+#root {
+  background-color: #69857154;
+  min-height: 100vh;
+}
+
+.App-header {
+  background-color: #282c34;
+  min-height: 10vh;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-left: 20%;
+  font-size: calc(10px + 2vmin);
+  color: #fff;
+}
+.App-main-paige {
```

Comment:
incearca sa fi consecvent in denumirea claselor. Citete putin de airBnb (e un guideline bun - combinatie intre BEM si OOCSS)

Reply:
Ok, multumesc, il voi studia.